bakery/service/hierarchy.go

```diff
@@ -13,7 +13,8 @@ import (
 // NewDockerHierarchy initializes new docker hierarchy.
 func NewDockerHierarchy() DockerHierarchy {
 	return &dockerHierarchy{imagesWithDependantsMap: make(map[string][]*DockerImage),
-		analyzedImagesFlatMap:             make(map[string]*DockerTreeItem),
+		analyzedImagesFlatMapDI:           make(map[string]*DockerImage),
+		analyzedImagesFlatMapDTI:          make(map[string]*DockerTreeItem),
```

Comment:
I don't think those `DI`/`DTI` in the names bring here any value... 
How about simplifying those names a bit?
`analyzedImagesFlatMapDI`  -> `images`
`analyzedImagesFlatMapDTI` -> `imagesTree`

Reply:
Sounds good